database/migrations/2021_12_12_171823_create_warnings_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWarningsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('warnings', function (Blueprint $table) {
+            $table->id();
+            $table->string('name',40);
```

Comment:
unique is missing

Reply:
Added unique to name column

Reply:
ok, it is ok to be migration for it. But it is still is not change of column, it is creation of unique index.